pom.xml

```diff
@@ -71,8 +77,18 @@
             <version>1.5.1</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
I had problems when running `mvn hpi:run` telling me I needed this, but I can remove it.